tests/unauthorized/userTest.js

```diff
@@ -1,287 +1,365 @@
+import { MatchersV3 } from '@pact-foundation/pact/v3'
+
 describe('Unauthorized: Currently testing user management,', function () {
   // CURRENT TIME
   var timeRightNow = new Date().getTime()
   var config = require('../config/config.json')
 
-  // LIBRARY INSTANCE
-  var oc
-
-  // PACT setup
-  const Pact = require('@pact-foundation/pact-web')
-  const provider = new Pact.PactWeb()
   const {
     invalidAuthHeader,
-    CORSPreflightRequest,
     capabilitiesGETRequestInvalidAuth,
-    pactCleanup,
     createOwncloud
   } = require('../pactHelper.js')
-  const { unauthorizedXmlResponseBody, origin } = require('../pactHelper.js')
+  const {
+    unauthorizedXmlResponseBody,
+    createProvider
+  } = require('../pactHelper.js')
 
-  const request = function (requestName, method, path) {
-    return {
-      uponReceiving: requestName,
-      withRequest: {
+  const request = async (provider, requestName, method, path, query) => {
```

Comment:
```suggestion
  const request = (provider, requestName, method, path, query) => {
```

Reply:
also the functionname request is a bit too generic, please change it to something like `addInvalidAuthInteraction`